tests/unit/datasets_tests/test_fer_2013.py

```diff
@@ -22,8 +23,9 @@
 def test_fer2013():
     dataset = FER2013()
     assert len(dataset.classes) == 7
-    assert dataset.available_subsets == ['train', 'test']
+    assert dataset.available_subsets == ['train', 'validation', 'test']
     assert len(dataset) == 1
     assert dataset[0][0].shape[0] == dataset.image_size
     assert dataset[0][0].shape[1] == dataset.image_size
-    assert dataset[0][1] == 0
+    assert dataset[0][0].shape[2] == 3
+    assert (dataset[0][1] == np.eye(dataset.num_classes)[0]).all()
```

Comment:
In general, it would be better to avoid using the same logic with the target code for generating expected values. In this case, let's use a literal array like `np.array([1, 0, 0, 0, 0, 0, 0])` instead.

Reply:
Thanks! I fixed it.